tests/colr_to_svg.py

```diff
@@ -38,60 +44,112 @@ def _svg_root(view_box: Rect) -> etree.Element:
     return svg_root
 
 
-def _colr_v0_to_svg(
+def _nice_path(svg_pen: svgPathPen.SVGPathPen) -> str:
+    # svgPathPen doesn't format #s nicely
```

Comment:
maybe we can change svgPathPen to our liking

Reply:
Works for me but beware that this means formatting svg the way I like it for little reason other than I like it that way :D